.gitignore

```diff
@@ -14,3 +14,5 @@ Carthage*
 **/TPPSecrets.swift
 fastlane/report.xml
 fastlane/*_output
+Cartfile
```

Comment:
We can't ignore `Cartfile` and `Cartfile.resolved`, and they already are in git

Reply:
nice catch, not sure how this got in